### ISSUE TITLE
[Suggestion] Disable 'Extended world selection' trial

### ISSUE THREAD

**Issue by HaXrBOT** [TheCSMods/mc-better-stats#132]
### What is this suggestion for?

Mod Itself

### Suggestion

I do not like how your 'Better Statistics Screen' mod is now being a platform to advertise for your other mods. At the very least i think there should be an option to completely turn it off in the config for use in modpacks. 

I don't like to be mean, so sorry if it comes off that way. But installing additional stuff like this without at least giving users the option first is a big nono for me.

### Code of Conduct

- [X] I agree that I will be available later for any follow-up questions to help with implementing the suggested thing.

**Comment by TheCSDev** [comment]
Hello, and thanks for the suggestion!  
I definitely understand your point, and I agree. I also did not intend on keeping it there for long, hence why it was finally removed about an hour ago (as I planned on having it there for like a day or so).

There is also [this GitHub issue from yesterday](https://github.com/TheCSMods/mc-better-stats/issues/131), about the same topic., and there I explained how to completely remove the button (as there is a config setting). However I do understand that even with that, the way to remove it isn't really presented in a clear way.

Apologies to you as well for the inconvenience and any complications that I may have caused. About an hour ago, another translation release was published, and EWS is now completely gone.  
The reason I embedded EWS in this mod in the first place is becuase unfortunately I don't have any other means of letting people know of anything I wanna let them know about. People sadly don't read changelogs and project descriptions, and they barely ever give feedback even when you keep asking them for it (unless of-course, they experience something inconvenient, and even then they just move on without giving feedback most of the time). So hence why, I figured the best to let people know about something would be to give a little preview demo of what it's like. Now of-course, I definitely understand how it can be seen as a bad practice despite all that.

Anyway thanks again for the suggestion and for sharing your thoughts with me. No worries about coming off as rude or anything, as I prefer and appreciate honesty, and I love getting honest feedback. This issue is also now resolved as well.
So yeah, have a great day as well!
